wgsl/index.bs

```diff
@@ -1202,29 +1205,6 @@ See https://github.com/gpuweb/gpuweb/issues/1534
   </xmp>
 </div>
 
-The first element in an array is at index 0, and each successive element is at the next integer index.
-See [[#array-access-expr]].
-
-An array element type must be one of:
-* a [=scalar=] type
-* a vector type
-* a matrix type
-* an [=atomic type|atomic=] type
-* an array type
-* a [=structure=] type
-
-Note: That is, the element type must be a [=plain type=].
-
-[SHORTNAME] defines the following attributes that can be applied to array types:
-* [=attribute/stride=]
-
-Restrictions on runtime-sized arrays:
```

Comment:
The first two restrictions are replaced by uses of "fixed footprint" in the definitions of both array and structure types.